### PR TITLE
Configure intercom to monitor the `.intercom-launcher` custom launcher

### PR DESCRIPTION
**Pull request by real34**
This PR configures the `intercom-launcher` class as a trigger to open the Intercom messenger and goes over every email contact occurrence to update it.

Can be tested on: https://deploy-preview-330--heuristic-almeida-1a1f35.netlify.app/

https://user-images.githubusercontent.com/75968/154533061-8f20b2ec-f99d-4379-ad92-4e737385090f.mp4

It updates every relevant contact call to action to open the messenger by default (and falls back to the `mailto:` link for users without JS or if intercom isn't loaded).

I've updated the fallback email address to target support@ (support) or hello@ (product) depending on the use case.

## Formatting

I noticed that some code was reformatted during git and/or editor hooks. I've reverted as many changes as possible, but some were a bit more tedious.

I'll try to followup with a PR allowing to have shared configs and tooling to get rid of such imtempestive changes and have a consistent formatting once and for all.

### CODE REVIEW

**Review by abailleulfc — APPROVED** [review]

File: source/docs/advanced/performance/overview.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ title: Overview
 
 In this section, you will learn advanced information about Performance in Front-Commerce.
 
-* [Preloading Routes' data](./preloading-routes.html)
```
Comment (mo-fc) [inline source/docs/advanced/performance/overview.md:8]: out of curiosity why are u changing the * to a -

File: source/docs/magento2/headless-payments.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Our Magento2 integration currently provides native adapters for the platforms be
 - [Payment on account](/docs/advanced/payments/payment-on-account.html)
 
 <blockquote class="info">
-  If you want to use a Payment module not yet listed above, please [`contact us`](mailto:contact@front-commerce.com) so we can provide information about a potential upcoming native support for it.
+  If you want to use a Payment module not yet listed above, please [`contact us`](mailto:hello@front-commerce.com) so we can provide information about a potential upcoming native support for it.
```
Comment (mo-fc) [inline source/docs/magento2/headless-payments.md:25]: missing the span

File: source/docs/magento2/overview.md [hunk 3]
```diff
@@ -36,4 +36,4 @@ We are still in the process of writing the documentation. But we will cover the
 - Interact with Magento 2's REST API
 - Disable Magento 2's storefront
 
-If you need more detailed information, please [contact us](mailto:contact@front-commerce.com). We will make sure to answer you in a timely manner.
+If you need more detailed information, please [contact us](mailto:support@front-commerce.com). We will make sure to answer you in a timely manner.
```
Comment (mo-fc) [inline source/docs/magento2/overview.md:39]: missing the span

**Review by mo-fc — CHANGES_REQUESTED** [review]

File: source/docs/advanced/performance/overview.md (hunk 1 above)
Comment (real34) [reply source/docs/advanced/performance/overview.md:8]: This was an automated formatting:

> I noticed that some code was reformatted during git and/or editor hooks. I've reverted as many changes as possible, but some were a bit more tedious.

File: source/docs/magento2/headless-payments.md (hunk 2 above)
Comment (real34) [reply source/docs/magento2/headless-payments.md:25]: Thanks!

File: source/docs/magento2/overview.md (hunk 3 above)
Comment (real34) [reply source/docs/magento2/overview.md:39]: Thanks!

**Pull request merged**